src/ffi/w3c/credential.rs

```diff
@@ -148,8 +149,8 @@ pub extern "C" fn anoncreds_credential_to_w3c(
             None => None,
         };
 
-        let w3c_credential =
-            credential_to_w3c(credential, cred_def.load()?.cast_ref()?, w3c_version)?;
+        let issuer_id = IssuerId::new(issuer_id).expect("Invalid issuer ID");
```

Comment:
I'm not a rust expert. Is this validated and handled correctly?

Reply:
If new returns an error it will panic. You can use `?` instead to not panic but pass along the error back over ffi.

Reply:
Thanks! Will update